### PR TITLE
ipatests: ipa-automountlocation command tests

### PR DESCRIPTION
**Pull request by menonsudhir**
This patch includes tests for ipa-automountlocation commands

### CODE REVIEW

File: ipatests/test_integration/test_nfs.py [hunk 1]
```diff
@@ -355,3 +355,330 @@ def verify_checksum_after_ipaclient_uninstall(
...
+        """
+        msg1 = "3 automount locations matched"
+        msg2 = "Location: {}".format(self.AUTOMOUNT_LOCATION1)
+        msg3 = "Location: {}".format(self.AUTOMOUNT_DEFAULT_LOCATION)
+        msg4 = "Location: {}".format(self.AUTOMOUNT_LOCATION2)
+        cmd = self.clients[0].run_command(
+            ["ipa", "automountlocation-find",
+             "--pkey-only"]
+        )
+        assert cmd.returncode == 0
```
Comment (amore17) [inline ipatests/test_integration/test_nfs.py:661]: This assert not required by default its asserting already.

**Comment by amore17** [comment]
IMO its suitable in xmlrpc tests.

**Review by amore17 — CHANGES_REQUESTED** [review]

**Comment by flo-renaud** [comment]
Hi @menonsudhir 
the tests should be added to the xmlrpc tests as they mainly test the CLI.
You need to check which ones are already available in ipatests/test_xmlrpc/test_automount_plugin.py and add only the missing ones.

**Comment by menonsudhir** [comment]
Closing this PR as this will be redone in xmlrpc tests.

**Pull request closed without merging**